### DISCUSSION TITLE
Release 0.1.2.pre as 0.1.2

### DISCUSSION THREAD

**Discussion by AndyObtiva** [kojix2/LibUI#86] [General]
Given that the latest `libui` gem version that is not `pre` is `0.0.15`, when someone tries to install `glimmer-dsl-libui` without allowing `pre` gems to be installed, they end up getting a very old version `0.5.24`, which was linked to `libui` `0.0.15`. 

To avoid this problem, I would like to suggest releasing version `0.1.2` as the same exact gem as `0.1.2.pre` given that it has been battle-tested and in use for a while and does not have issues.

I would avoid releasing `0.1.3.pre` as that version has issues at the moment, which were reported at the upstream C libui project and have not been fixed it: https://github.com/libui-ng/libui-ng/issues/252 (and corresponding https://github.com/AndyObtiva/glimmer-dsl-libui/issues/61)

**Comment by kojix2** [comment]
That makes sense.

**Reply by kojix2** [reply]
Hopefully I can tackle this next Saturday. Although, I will simply release it as 0.1.2 with the same content as 0.1.2.pre.
I am also interested in https://github.com/libui-ng/libui-ng/pull/261.
I'm going to build and test this one myself, since cody says they don't have enough testers.

**Comment by rubyFeedback** [comment]
In my own limited experience, I found working with non-pre gems much easier, so I found it better to just release versioned gems (and there, more versions in general, as a consequence), rather than deal with pre-releases. I just found it simpler that way.

**Comment by kojix2** [comment]
As promised, I released v0.1.2. It is exactly the same as v0.1.2.pre. Continuously updating the software version is truly challenging. Yes, but our journey continues...

Let's close this discussion. As always, I believe it's far from perfect, but if any new issues arise, please open a new issue or discussion.
